src/app/files.js

```diff
@@ -107,6 +109,20 @@ function Files (storage) {
     return files
   }
 
+  // @TODO: use this API
+  this.add = function addFile (filepath, content) {
```

Comment:
Why does this needs to be added here? I would prefer to keep it separate or only add the current method `createNonClashingName`.